app/assets/stylesheets/alchemy/image_library.scss

```diff
@@ -140,3 +217,53 @@ $image-overlay-form-width: 350px;
     }
   }
 }
+
+.picture-overlay-navigation {
+  position: absolute;
+  top: 50%;
+  left: 0;
+  width: 100%;
+
+  .previous-picture,
+  .next-picture {
+    position: absolute;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets<br>Properties should be ordered cursor, height, line-height, margin-top, position, text-align, text-decoration, top, width